### PR TITLE
Styling of sidebar & alpha label

### PR DESCRIPTION
**Pull request by kkuepper**
The goal is to get closer to the design in Figma.

Some things I'm struggling with:
- The scrollbars should be hidden.
But that requires custom CSS and I don't know where to put it or if there is a better solution
- The alpha label requires custom padding values
- "Playlists" in the sidebar should have font-size: 15px and padding-bottom: 2px
- Icon nav.search has fill mixed up

@sifferhans can you help me with addressing those issues?

### CODE REVIEW

**Comment by kkuepper** [comment]
You can check out how it looks: https://salmon-pebble-009136d03-141.westeurope.2.azurestaticapps.net/

@sclausendk Maybe you can comment on what else is off.

**Comment by sifferhans** [comment]
Sure, I can take a look!

File: components/sidebar/SidebarElement.vue [hunk 1]
```diff
@@ -5,20 +5,21 @@ const links: {
   title: string;
   link: RoutesNamedLocations;
 }[] = [
-  { title: "Home", link: { name: "index" } },
-  { title: "Browse", link: { name: "browse" } },
-  { title: "Search", link: { name: "search" } },
+  { title: "Home", link: { name: "index" }, icon: "nav.home" },
```
[the remark this answers is not in the document]
Comment (kkuepper) [reply components/sidebar/SidebarElement.vue:8]: Nevermind. I misunderstood the declaration

[the remark this answers is not in the document]
Comment (SimonSimCity) [reply components/sidebar/SidebarElement.vue:9]: I thought of getting typing in here so you can see which icons do exist and so on, I'll create a draft for this in a later PR.

**Review by SimonSimCity — APPROVED** [review]

**Comment by sifferhans** [comment]
> The alpha label requires custom padding values

What do you mean here @kkuepper ?

**Review by sifferhans — APPROVED** [review]
lgtm

**Pull request merged**